### PR TITLE
BOM-1000 Python3 Compatibility

### PR DESCRIPTION
**Pull request by Ayub-Khan**
Current python3 tests Status:
Running Command : 'paver e2e_test --exclude="whitelabel\|enterprise"'
All tests passing locally.
Tests run using jenkins so we will have to create a new job to run tests on github for py3.

### CODE REVIEW

File: regression/tests/studio/test_course_pages.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+"""
```
Comment (awais786) [inline regression/tests/studio/test_course_pages.py]: is this new file ?

**Comment by awais786** [comment]
@Ayub-Khan only question about newly added test. PR seems fine.

**Comment by jmbowman** [comment]
The change that caused the microsites failure has been reverted on master, so a rebase should now fix that.

File: pylintrc [hunk 2]
```diff
@@ -80,7 +80,9 @@ disable =
 	too-many-arguments,
 	too-many-locals,
 	unused-wildcard-import,
-	duplicate-code
+	duplicate-code,
+	unicode-format-string,
+	super-on-old-class
```
[the remark this answers is not in the document]
Comment (Ayub-Khan) [reply pylintrc]: ok.

File: pylintrc [hunk 3]
```diff
@@ -180,4 +459,4 @@ int-import-graph =
 [EXCEPTIONS]
 overgeneral-exceptions = Exception
 
-# a795e1dfab29b0fe16d892e7a0ee736ec2927408
+# a78cbfddf357eb444028233318f94b1c8c343447
```
Comment (awais786) [inline pylintrc]: what is this ?

File: regression/tests/whitelabel/test_user_account.py [hunk 4]
```diff
@@ -77,7 +70,7 @@ def test_reset_password(self):  # TE-2044
         self.login_page.visit()
         self.login_page.send_forgot_password(self.user_email)
         self.assertTrue(
-            self.login_page.is_password_reset_email_message_visible()
+            self.login_page.is_password_reset_email_message_visible
```
Comment (awais786) [inline regression/tests/whitelabel/test_user_account.py]: is this by mistake ?

File: pylintrc (hunk 3 above)
Comment (Ayub-Khan) [reply pylintrc]: This is auto generated file using the edx-lint command 
`edx_lint write pylintrc`

File: regression/tests/whitelabel/test_user_account.py (hunk 4 above)
Comment (Ayub-Khan) [reply regression/tests/whitelabel/test_user_account.py]: is_password_reset_email_message_visible is property.

**Pull request merged**